docs/source/quicktour.mdx

```diff
@@ -211,220 +211,93 @@ and get tensors back. You can specify all of that to the tokenizer:
-keys directly to tensors, for a PyTorch model, you need to unpack the dictionary by adding `**`.
-
-```py
->>> pt_outputs = pt_model(**pt_batch)
-===PT-TF-SPLIT===
->>> tf_outputs = tf_model(tf_batch)
-```
+## Using the AutoModel
 
-In 🤗 Transformers, all outputs are objects that contain the model's final activations along with other metadata. These
-objects are described in greater detail [here](main_classes/output). For now, let's inspect the output ourselves:
+🤗 Transformers provides a simple and unified way to load pretrained instances, which means you can load an [`AutoModel`] like you would load an [`AutoTokenizer`]. The only difference is selecting the correct [`AutoModel`] for the task. Since we are doing text - or sequence - classification, load the [`AutoModelForSequenceClassification`]. The TensorFlow equivalent is simply [`TFAutoModelForSequenceClassification`]:
 
-```py
->>> print(pt_outputs)
-SequenceClassifierOutput(loss=None, logits=tensor([[-4.0833,  4.3364],
-        [ 0.0818, -0.0418]], grad_fn=<AddmmBackward>), hidden_states=None, attentions=None)
+```python
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+>>> pt_model = AutoModelForSequenceClassification.from_pretrained(model_name)
 ===PT-TF-SPLIT===
->>> print(tf_outputs)
-TFSequenceClassifierOutput(loss=None, logits=<tf.Tensor: shape=(2, 2), dtype=float32, numpy=
-array([[-4.0833 ,  4.3364  ],
-       [ 0.0818, -0.0418]], dtype=float32)>, hidden_states=None, attentions=None)
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+>>> # This model only exists in PyTorch, so we use the _from_pt_ flag to import that model in TensorFlow.
+>>> tf_model = TFAutoModelForSequenceClassification.from_pretrained(model_name, from_pt=True)
 ```
 
-Notice how the output object has a `logits` attribute. You can use this to access the model's final activations.
-
 <Tip>
-
-All 🤗 Transformers models (PyTorch or TensorFlow) return the activations of the model *before* the final activation
-function (like SoftMax) since this final activation function is often fused with the loss.
-
+See the [task summary](./task_summary) for which [`AutoModel`] to use for which task.
 </Tip>
 
-Let's apply the SoftMax activation to get predictions.
+Now you can pass your preprocessed batch of inputs directly to the model. If you are using a PyTorch model, unpack the dictionary by adding `**`. For TensorFlow models, pass the dictionary keys directly to the tensors:
 
-```py
->>> from torch import nn
->>> pt_predictions = nn.functional.softmax(pt_outputs.logits, dim=-1)
+```python
+>>> pt_outputs = pt_model(**pt_batch)
 ===PT-TF-SPLIT===
->>> import tensorflow as tf
->>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
+>>> tf_outputs = tf_model(tf_batch)
 ```
 
-We can see we get the numbers from before:
+The model outputs the model's final activations in the `logits` attribute. Apply the softmax activation to the `logits` to retrieve the predictions:
```

Comment:
```suggestion
The model outputs the model's final activations in the `logits` attribute. Apply the softmax function to the `logits` to retrieve the probabilities:
```
Softmax is not an activation, and it returns probabilities. Predictions would be the classes with the maximum probabilities/logits.